docs/src/rsh/step/index.md

```diff
@@ -463,7 +463,23 @@ It accepts an optional bytes argument, which is included in any reported violati
 closeTo( Who, after, nonNetPayAmt )
 ```
 
-@{ref("rsh", "closeTo")} Has participant `{!rsh} Anybody` make a publication, then transfer the `{!rsh} balance()` and the non-network pay amount to `{!rsh} Who` and end the DApp after executing the function `{!rsh} after` in a step.
+@{ref("rsh", "closeTo")} Has participant `{!rsh} Anybody` make a publication, then transfer the `{!rsh} balance()` and the non-network pay amount to `{!rsh} Who` and end the dApp after executing the function `{!rsh} after` in a step.
```

Comment:
:( We say "DApp" all over the place, so if you want to try and convince me to change that, you'll have to do that separately than this

Reply:
That's not a hill I'm willing to die on at this point in time.